tools/meteor-services/deploy.js

```diff
@@ -321,6 +334,122 @@ function canonicalizeSite(site) {
   return parsed.hostname;
 };
 
+// Executes the poll to check for deployment success and outputs proper messages
+// to user about the status of their app during the polling process
+function executePollForDeploymentSuccess(versionId, deployPollTimeout, result) {
+  // Create a default polling configuration for polling for deploy / build
+  // In the future, we may change this to be user-configurable or smart
+  // The user can only currently configure the polling timeout via a flag
+  let pollingState = new PollingState(deployPollTimeout);
+  
```

Comment:
line has trailing whitespace (you should be able to configure your editor to trim trailing whitespace on save?). There are other lines with trailing whitespace in the file as well. Too bad GitHub doesn't highlight it.

Reply:
Added to IDEs 👍